k8s/templates/_helpers.tpl

```diff
@@ -81,8 +81,6 @@ Generate the shared environment variables between studio app and workers
   value: "yes"
 - name: RUN_MODE
   value: k8s
-- name: DATA_DB_HOST
```

Comment:
We remove `DATA_DB_HOST` in here, since they're going to be individually defined in each deployment.